autogpt/config/config.py

```diff
@@ -159,17 +159,17 @@ def __init__(self) -> None:
         self.plugins: List[AutoGPTPluginTemplate] = []
         self.plugins_openai = []
 
-        plugins_allowlist = os.getenv("ALLOWLISTED_PLUGINS")
-        if plugins_allowlist:
-            self.plugins_allowlist = plugins_allowlist.split(",")
+        plugins_allowed = os.getenv("ALLOWED_PLUGINS")
```

Comment:
`plugins_allowed` is a bit of an ambiguous name: is it a `bool` indicating whether plugins are allowed, or a `list` of allowed plugins?